src/Backend/Modules/Groups/Installer/Installer.php

```diff
@@ -68,7 +68,7 @@ private function insertDashboardSequence()
     public function install()
     {
         // load install.sql
-        $this->importSQL(dirname(__FILE__) . '/Data/install.sql');
+        $this->importSQL(dirname(__FILE__) . '/Data/install.sql'); // @todo Use schema create tool
```

Comment:
This rule can be deleted.
Since ->addEntitiesInDatabase() has done this already.